README.md

```diff
@@ -62,3 +62,9 @@ Start using the app:
 
 Note that it is necessary to keep the structure of the archive intact: the scripts in `bin` rely
 on the relative path to other files in the archive.
+
+[ubuntu-build]: https://github.com/SpineEventEngine/ProtoData/actions/workflows/build-on-ubuntu.yml
```

Comment:
Jekyll Front Matter is not supported for README.md files under GitHub. So we cannot have a page-wide variables followed by Liquid expansion.